features/signal/external/feature.java

```diff
@@ -0,0 +1,53 @@
+package signal.external;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.sdkfeatures.Feature;
+import io.temporal.sdkfeatures.Run;
+import io.temporal.sdkfeatures.Runner;
+import io.temporal.workflow.SignalMethod;
+import io.temporal.workflow.Workflow;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@WorkflowInterface
+@ActivityInterface
```

Comment:
No activities in here. There's a fix in my PR that allows init to work without this annotation / no activities.

Reply:
Stole it. Whoever lands first wins?

Reply:
I stole a Go fix from his review too and landed it. Sorry @Sushisource if I cause a conflict!